src/main/webapp/js/pages/index.js

```diff
@@ -34,10 +49,35 @@ document.querySelector('.course-list').addEventListener('click', () => {
       course => {sections2[course] = courseInfo[course].sections[1]});
   const schedules = [sections1, sections2];
 
-  // TODO(ramyabuva): add pagination for multiple schedules
-  selected.forEach(course => {
-    Calendar.addCourse(courseInfo[course], schedules[0][course]);
-  });
+  // by default, add the first schedule to the calendar after response received
```

Comment:
This is one of the smallest nits possible, but general style guide rules are that comments should usually be proper English sentences.

Specifically: capitalize the start of your comments, and end them with a period. Same for other comments below.